test-framework/junit5/src/main/java/io/quarkus/test/junit/NativeTestExtension.java

```diff
@@ -175,6 +179,39 @@ public void postProcessTestInstance(Object testInstance, ExtensionContext contex
         }
     }
 
+    /**
+     * Since {@link TestResourceManager} is loaded from the ClassLoader passed in as an argument,
+     * we need to convert the user input {@link QuarkusTestProfile.TestResourceEntry} into instances of
+     * {@link TestResourceManager.TestResourceClassEntry}
+     * that are loaded from that ClassLoader
+     */
+    private List<TestResourceManager.TestResourceClassEntry> getAdditionalTestResources(
```

Comment:
This code is the same as in QuarkusTestExtension: https://github.com/quarkusio/quarkus/blob/a5c0d7710d677462a92855ba5948c2876f18260e/test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java#L435

As the logic inside is not very complicated, I decided to not refactor both extensions and reuse the same method, but let me know if you prefer me doing so.